### ISSUE TITLE
loss status of the model has been trained good?

### ISSUE THREAD

**Issue by LearnerInGithub** [dengdan/seglink#9]
@dengdan  Hello! Thanks for your kindly sharing your repo here, now while I trained your model, I find the loss stuck  in the range [3.5, 4.5] for a long time, does it normal?  my learning rate is 0.001, trained on ICDAR2015.

**Comment by dengdan** [comment]
No, it's not normal. The model might get stuck in a bad basin unluckily sometimes. You can restart your training from an earlier point.  The training process ends with loss value  around 1.0.

**Comment by jiangli1679** [comment]
Hi, @dengdan I trained the model on SynthText dataset with pretrained SSD model, the loss stuck around 2.8, does it normal? My learning rate is 0.0001 and optimizer is Adam. Thanks.

**Comment by jiangli1679** [comment]
@dengdan @LearnerInGithub 
Hi, I trained the model on SynthText dataset with pretrained SSD model, the loss stuck around 2.8, does it normal? My learning rate is 0.0001 and optimizer is Adam. Thanks.
